socialhome/templates/base.html

```diff
@@ -102,6 +102,7 @@
         <script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
         <!-- Compiled project JS -->
         <script src="{% static 'js/project.js' %}"></script>
+        <script src="{% static 'js/webpack.bundle.js' %}"></script>
```

Comment:
Please add this as a super call in the template loading the streams app, like indicated in https://github.com/jaywink/socialhome/pull/193#issuecomment-317281378. That way it's only loaded on pages it needs to be.